app/models/category.rb

```diff
@@ -121,14 +122,14 @@ class Category < ActiveRecord::Base
   # Allows us to skip creating the category definition topic in tests.
   attr_accessor :skip_category_definition
 
-  @topic_id_cache = DistributedCache.new('category_topic_ids')
+  has_distributed_cache :topic_id_cache, 'category_topic_ids'
```

Comment:
I think just name this `distributed_cache` no need for the `has_` cause it can be a bit confusing, people may think it is active recordy (has_one , has_many)